plugins/intel-me/meson.build

```diff
@@ -0,0 +1,16 @@
+if get_option('plugin_amt').disable_auto_if(host_machine.system() != 'linux').allowed()
```

Comment:
presumably you want to be renaming the plugin option from `plugin_amt` now

Reply:
I wasn't sure; can we include it as a "deprecated" name to not break things like openbmc?

Reply:
I don't know if deprecated works for renaming things.  However openbmc won't break.  They compile with `-Dauto_features=disabled`

Reply:
Agree, fixed.